data/conf.d/10-smartmontools.cfg

```diff
@@ -0,0 +1,2 @@
+[smart]
+sonames=libbd_smartmontools.so.3
```

Comment:
This was confusing. I've removed the file and instead set priorities in the default config file.

Reply:
FYI, the priorities only work with the config file present, otherwise it's hardcoded for a single plugin chosen during build time. I.e. distributors wanting to have smartmontools plugin by default will have to disable libatasmart or always ship the config file.